packages/create-remix/copy-template.ts

```diff
@@ -384,10 +384,14 @@ function isValidGithubRepoUrl(
 }
 
 function isGithubRepoShorthand(value: string) {
+  if (isUrl(value)) {
+    return false;
+  }
```

Comment:
I don't _think_ we'd get collisions for URLs with file extensions - but it felt safe to ensure that we short circuit exit on non-RULs when looking for a shorthand